src/Arcus.EventGrid.Tests.Unit/Publishing/EventGridPublisherBuilderTests.cs

```diff
@@ -160,10 +155,10 @@ public void WithExponentialRetryWithCircuitBroker_PositiveExceptionsAllowedBefor
         {
             // Arrange
             const int positiveInt = 10;
-            TimeSpan positiveInterval = TimeSpan.FromSeconds(5);
+            var positiveInterval = TimeSpan.FromSeconds(5);
 
             // Act
-            IEventGridPublisher publisher =
+            var publisher =
                 CreateEventGridBuilder()
```

Comment:
is it the diff or is this accidently moved to a newline?

Reply:
It's the diff